docs/api/datetime.md

```diff
@@ -273,6 +275,16 @@ Developers can provide their own buttons for advanced custom behavior.
 
 <CustomizingButtons />
 
+## Multiple Date Selection
+
+If the `multiple` property is set to `true`, multiple dates can be selected from the calendar picker at once. Clicking a selected date will deselect it.
```

Comment:
~ The "at once" bit stuck out to me as possibly being confusing to the multiple feature. While technically you can select multiple dates at once with manually providing an array of values, the follow-up sentence makes me feel like it implies while clicking/interacting with the calendar picker.

Thoughts around omitting that piece and having it read similar to:
> If the `multiple` property is set to `true`, multiple dates can be selected from the calendar picker. Clicking a selected date will deselect it.

Reply:
Works for me 👍